### PR TITLE
Some cleaning

### PR DESCRIPTION
**Pull request by rofrol**

### CODE REVIEW

File: src/Counter.elm [hunk 1]
```diff
@@ -12,9 +12,11 @@ type alias Model =
   , btnClicks : Int
   }
 
+init : Int -> Model
 init num =
   Model num 0
 
+view : String -> { c | btnClicks : a, num : b } -> Html Msg
```
Comment (afcastano) [inline src/Counter.elm:19]: `view: String -> Model -> Html Msg` would also work here, wouldn't it?

**Comment by afcastano** [comment]
Man... Thank you very much!!! I am learning a lot from your changes.

Let me know when you finish and I'll merge it. 

PS: I left a comment in one of them.

Thanks again!

**Comment by afcastano** [comment]
Thanks!

**Comment by rofrol** [comment]
I thank you for your code. I was just trying to understand this code, that's why I was doing this cleaning :)

**Pull request merged**